Code/EditorPlugins/Assets/EditorPluginAssets/PropertyAnimAsset/PropertyAnimObjectManager.cpp

```diff
@@ -37,21 +37,3 @@ ezStatus ezPropertyAnimObjectManager::InternalCanMove(
     return ezStatus("The structure of the context cannot be animated.");
   return ezStatus(EZ_SUCCESS);
 }
-
-bool ezPropertyAnimObjectManager::IsTemporary(const ezDocumentObject* pObject) const
```

Comment:
why has this been removed ?

Reply:
It has been moved to ezDocumentObjectManager